src/main/java/com/iprody/user/profile/userprofileservice/config/ObservedAspectConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package com.iprody.user.profile.userprofileservice.config;
```

Comment:
propagate all over the solution
```suggestion
package com.mykytaaa.user.profile.userprofileservice.config;
```